sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/MainActivity.java

```diff
@@ -88,4 +106,30 @@ protected void onCreate(Bundle savedInstanceState) {
 
     setContentView(binding.getRoot());
   }
+
+  private void captureExceptionWithAttachments() {
+    Sentry.withScope(
```

Comment:
L: do we need to use a `Sentry.withScope`? we try to avoid them on Mobile at least

Reply:
I just want to send the attachment with the exception and not all events. If I add the attachment to the global scope it is going to be sent with every event. I'm also fine with doing that.

Reply:
maybe adding a comment, because that means none of the other Scope fields will be sent either, if people just copy and paste, other fields will be missing.

Reply:
I removed the Thread and just create a small file.